source/GDPR/index.rst

```diff
@@ -85,3 +85,7 @@ Deletion will take place via a standard GLPI automatic action that can be config
    :scale: 65 %
 
 
+FAQ
+---
+
+If you have any questions about using the plugin, please consult `our faq <https://faq.teclib.com/04_Plugins/GDRP/>`__
```

Comment:
```suggestion
If you have any questions about using the plugin, please consult `our FAQ <https://faq.teclib.com/04_Plugins/GDRP/>`__
```

FAQ should be uppercase since it is an initialism.
https://www.grammarly.com/blog/abbreviations/

This replacement suggestion applies for the rest of the files where this was added.